components/Navbar.js

```diff
@@ -21,16 +38,54 @@ const Tab = ({ href, children }) => {
   )
 }
 
-const NavbarStyled = styled.nav.attrs({
-  className: 'flex items-center bg-white',
-})``
-
-const Navbar = () => (
-  <NavbarStyled>
+const Tabs = () => (
+  <>
     <Tab href="/">Support</Tab>
     <Tab href="/self-care">Self-Care</Tab>
     <Tab href="/stories">Stories</Tab>
-  </NavbarStyled>
+  </>
 )
 
+const NavbarStyled = styled.nav.attrs(
+  ({ left, right, border, colour = 'white' }) => ({
+    className: `flex items-center justify-between bg-${colour}${
+      border ? ' border-b border-lightgray' : ''
+    }${left ? ' pl-4.5' : ''}${right ? ' pr-5.5' : ''}`,
+  }),
+)``
+
+const Navbar = ({ border, links, title, heart, back, empty, colour }) => {
+  if (links)
+    return (
+      <NavbarStyled>
+        <Tabs />
+      </NavbarStyled>
+    )
+
+  return (
+    <NavbarStyled
+      left={back}
+      right={heart || empty}
+      colour={colour}
+      border={border}
+    >
+      {back && <Back />}
+      {title && <Title>{title}</Title>}
+      {heart && <Heart />}
+      {empty && <div />}
+    </NavbarStyled>
+  )
+}
+
+export const getNavbarOptions = ({ _type, title }) => {
+  switch (_type) {
+    case 'selfcareTechnique':
+      return { border: true, back: true, heart: true, title, colour: 'coral' }
+    case 'questionnaire':
```

Comment:
empty case?

Reply:
`case 'questionnaire':` doesn't go anywhere - thinking about it now I guess this is cos we don't have a questionnaire yet

Reply:
This is the syntax for getting multiple cases to run the same line of code. In this case it will run line 86 — and return an identical navbar to 'page' :)